### PR TITLE
Merging Ava branch to dev

### LINKED ISSUE #239: Laptop explodes when opening R
* **Description of bug**

Laptop go boom when opening R.

* **Tell us how to reproduce the error**

1. Open R
2. Explode

* **Expected behavior**

Laptop does not explode.

* **Platform**

Were you running the code on:

- [X] Local R
- [ ] Citrix
- [ ] Non-network laptop

* **Screenshots**

R

### PR DESCRIPTION
**Pull request by avasharma**
## Proposed changes

It adds my extremely important narrative changes to the text file.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [X] Documentation Update (change to naming or other documentation)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR._

- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)
- [X] I have checked that my changes have not broken any other functionality
- [ ] I have linked to any issues this PR fixes 

## Points for review

Add checkboxes here for any specific aspects of your PR you would like to be checked in peer review. If you don't specify anything here, the reviewer will check:

- [ ] All of your unit tests pass
- [X] Your code is clear and easy to understand
- [X] You have documented any new features
- [ ] They can run the code you have written
- [X] Your changes do not break any other functionality

## Who is reviewing this PR?

@DIPAD-Fran-Bryden @ManiScriptWizard

### CODE REVIEW

File: text_file.txt [hunk 1]
```diff
@@ -2,4 +2,6 @@ This is a random text file. You can add and remove things to it to test how git
 
 
 This is some text.
-I am just normally editing a text file here.
+I am confused...
+
```
Comment (DIPAD-Fran-Bryden) [inline text_file.txt:6]: Unnecessary whitespace here

File: text_file.txt [hunk 2]
```diff
@@ -2,4 +2,6 @@ This is a random text file. You can add and remove things to it to test how git
...
+collaboration from Ben :)
```
Comment (DIPAD-Fran-Bryden) [inline text_file.txt:7]: Good collaboration

**Review by DIPAD-Fran-Bryden — APPROVED** [review]
Good code overall

**Review by ManiScriptWizard — APPROVED** [review]
looks good
